themes/custom/lex/scss/regions/_footer.scss

```diff
@@ -13,6 +13,22 @@ body footer {
     @include screens-below(sm) {
       padding: 2rem 3rem 0;
     }
+    nav.contextual-region {
+      flex: 0 0 30%;
+      @include screens-below(sm) {
+        flex: 100%;
+      }
+      & > ul {
+        column-count: 2;
+        height: 125px;
+        @include screens-below(sm) {
```

Comment:
This is what you were talking about with the built in way to quickly handle breakpoints, right?

Reply:
Custom function but yes that's what I'm talking about.